dsl/meta.go

```diff
@@ -63,6 +63,22 @@ import (
 //        })
 //    })
 //
+// - "struct:field:type" overrides the Go struct field type specified in the design, with one caveat;
+//		if the type would have been a pointer (such as its not Required) the new type will also be a pointer.
```

Comment:
Fix indentation on the entire comment block. Godoc seems to show these comments a bit off.